students/thorn/lesson06/mailroom_6.py

```diff
@@ -0,0 +1,179 @@
+import os
+
+"""
+Thomas Horn
+mailroom.py
+Tracks donor information and automates the sending of thank you notes.
+"""
+donors = {
+    'Tom Horn': [599.23, 1000.00],
+    'Theo Hartwell': [0.01, 0.01, 0.1],
+    'Bailey Kimmitt': [8723.22, 27167.22, 91817.66],
+    'Paul Hubbell': [90012.32, 2312.24],
+    'David Beckham': [1817266.11, 123123.66, 111335.112]
+}
+
+def get_donor_names():
+    """ Returns the donor keys - representing the donor names. """
+    return donors.keys()
+
+
+def get_donor_totals(donor):
+    """ Returns the total donation for a donor.  Not currently active. Except for testing. """
+    print(sum(donors[donor]))
+    return(sum(donors[donor]))
```

Comment:
I would get rid of this unimplemented function and the test, since it's basically testing the built-n `sum()`